ethical-advertising-update-2019.rst

```diff
@@ -0,0 +1,117 @@
+  We continue to believe that advertising can be well targeted without tracking users.
+* We launched an advertising vertical specific to
+  :doc:`promote open jobs <lessons-from-hiring-manager-interviews>` to our developer audience.
+
+.. _community ads program: https://docs.readthedocs.io/en/latest/advertising/ethical-advertising.html#community-ads
+
+Our ethical advertising approach, with no user tracking, has been key to our sustainability
+and has allowed us to continue to build new features and give back to our community.
+
+
+Our new ad format
+-----------------
+
+One type of ad we've wanted to try but it took a while to get it right was text-only ads.
+From the start, we were very keen on text-only advertisements
+because they aren't as obtrusive as image-based ads
+and they are easier for advertisers to create and experiment with multiple variations.
+What we didn't know was whether users would engage with them at similar rates to our other ads.
+
+We ran a small experiment with a text-only house ad
+on the Read the Docs documentation itself and the results were very promising.
+Engagement rates were over double our site average
+and even compared favorably with the same ad in the sidebar including an image.
+
+I'd like to thank Eric Berry over at `CodeFund <https://codefund.io/>`_ for some great ideas
+and insights around this kind of ad format.
+
+.. figure:: img/2019-text-only-ad.png
+   :alt: A text-only ad on the Read the Docs documentation. Yes, this new format will be blocked by most ad blockers.
+   :width: 80%
+
+   A text-only footer advertisement on the Read the Docs documentation
+
+Currently, we only show advertising on a :doc:`subset of Sphinx themes <ads-on-other-themes>`
+and about 30% of our site traffic does not get an ad.
+We purposely did this slowly to make sure our ads look integrated with Read the Docs and less obtrusive to users.
+**This new ad format will allow us to show advertising on all documentation themes.**
+
+We plan to roll this new ad format out more widely in stages:
+
+* To start with, we are going to enable this ad format for community and house advertising
+  on pages that already have an ad on them.
+  As always, we never display two ads on the same page so this ad will be instead of any other ad.
+* After that, we will allow paid advertisers to use this new format.
+* We will email users of custom documentation themes to let them know this change is coming.
+* At first, we will show only community and house ads on these custom themes.
+* Finally, we'll turn on our normal ad rotation to all themes. We expect to complete this by July.
+
+We will closely monitor feedback from document authors, readers, and advertisers during this process.
+If you have questions or comments on this, please `let us know`_.
+
+.. _let us know: mailto:ads@readthedocs.org?subject=New+Ad+Format+Feedback
+
+Users can opt-out of paid advertising themselves and for any projects they maintain
```

Comment:
I think with the latest changes it's not possible to remove ads for a project anymore.

Reply:
They can't remove all ads without speaking to us but they can remove paid ads. For example, we could remove paid ads from the "docs" project here: https://readthedocs.org/dashboard/docs/advertising/